dev/devicelab/bin/tasks/flavors_test_win.dart

```diff
@@ -5,13 +5,38 @@
 import 'package:flutter_devicelab/framework/devices.dart';
```

Comment:
This file is identical to `flavors_test.dart`.

Can you delete this file and change 
https://github.com/flutter/flutter/blob/44ecbbcc54c1029eed51662dc7f123607384b564/.ci.yaml#L4544
to 
```
       task_name: flavors_test
```
I _think_ that should work?

Reply:
It worked :)